dfusion_rust_core/src/models/standing_order.rs

```diff
@@ -57,6 +57,8 @@ impl From<mongodb::ordered::OrderedDocument> for StandingOrder {
                 .iter()
                 .map(|raw_order| raw_order.as_document().unwrap())
                 .map(|order_doc| super::Order {
+                        slot: order_doc.get_str("slot").unwrap().parse().unwrap(),
```

Comment:
Can you check the representation inside mongo? Standing orders don't have a `slot` or `slot_index` field so you should be crashing here (because of the unwrap). Maybe we should make slot and slot_index an Optional so that it can be None?